### PR TITLE
Split dist/leaflet.css into separate files, build&minify them

### PR DESCRIPTION
**Pull request by IvanSanchez**
This splits `dist/leaflet.css` into a lot of smaller files, then `build/build.js` concatenates them (sourcemaps and all) and uses [cssnano](http://cssnano.co/) to minify the CSS.

This results in a size savings of around ~2.5KB when using the minified CSS.

The examples in `debug/` are also tweaked so the individual CSS files are used.

### CODE REVIEW

File: debug/leaflet-include.js [hunk 1]
```diff
@@ -37,7 +37,12 @@
 
 	var path = getSrcUrl();
     for (var i = 0; i < scripts.length; i++) {
-		document.writeln("<script src='" + path + scripts[i] + "'></script>");
+		if (scripts[i].substr(-3) === '.js') {
+			document.writeln("<script src='" + path + scripts[i] + "'></script>");
+		}
+		if (scripts[i].substr(-4) === '.css') {
```
Comment (hyperknot) [inline debug/leaflet-include.js]: Great work, will help a lot navigating the CSS files. Maybe rename `scripts` here to `assets` or `sources` now they are not scripts?

Comment (IvanSanchez) [inline debug/leaflet-include.js]: Feel free to add commits to this PR to tweak variable names and the like. It's not that important for me :-P

Comment (hyperknot) [inline debug/leaflet-include.js]: done

**Comment by mourner** [comment]
Nice work, although I'm not sure I like the overhead of tons of CSS files scattered among different folders... I think the current leaflet.css is small enough to be much easier to maintain as a single file, and 2.5k savings for CSS are pretty much negligible.

**Comment by cherniavskii** [comment]
@IvanSanchez @mourner should we close this?

**Pull request closed without merging**